CMakeLists.txt

```diff
@@ -1217,17 +1217,26 @@ else ()
   set(HAVE_HALF_FLOAT OFF)
 endif ()
 
+set(BUILD_PT_TRACER OFF)
+set(BUILD_PT_POST_PROCESSOR OFF)
 set(proc_supports_pt OFF)
+# Right now we only build PT related libraries on Linux x86_64.
 if (LINUX AND X86 AND X64)
-  # Right now we only build libipt on Linux x86_64.
-  # We define a variable to indicate whether DR will build libipt.
-  # TODO i#5505: In the next PR, we need to check whether the libipt submodule is initialized
-  # and print a warning message when the libipt directory is empty.
-  set(HAVE_LIBIPT ON)
+  # We will always build the PT tracer no matter whether the system supports PT.
```

Comment:
Add to the comment the rationale behind that: users may want to use the built binaries on a different system that does support PT.

Reply:
Done.